nix/cardano/packages/haskell.nix

```diff
@@ -181,7 +182,7 @@ in
           packages.cardano-ledger-byron.components.tests.cardano-ledger-byron-test = {
             preCheck = ''
               export CARDANO_MAINNET_MIRROR="${byron-chain}/epochs"
-              cp ${../environments/mainnet/byron-genesis.json} ./mainnet-genesis.json
+              cp ${iohk-nix}/cardano-lib/mainnet/byron-genesis.json ./mainnet-genesis.json
```

Comment:
🤔 should all of these be exported from `iohk-nix` so that you don't need to reach into he `iohk-nix` file structure?

Reply:
Maybe if it helps organization elsewhere.  No strong preference either way.